app/components/rdf-input-fields/instance-multi-selector.js

```diff
@@ -32,11 +34,10 @@ export default class RdfInstanceMultiSelectorComponent extends InputFieldCompone
     const instanceLabelProperty = this.getFormProperty('instanceLabelProperty');
     const instanceApiUrl = this.getFormProperty('instanceApiUrl');
 
-    const response = await fetch(`${instanceApiUrl}?page[size]=${PAGESIZE}`, {
-      headers: {
-        Accept: 'application/vnd.api+json',
-      },
-    });
+    const response = await fetch(
+      `${instanceApiUrl}?page[size]=${PAGESIZE}`,
+      ACCEPT_HEADER
```

Comment:
i feel like this may be too much, i would just use the value as the constant

Reply:
since all headers in this file are the same you could make that a local constant if you'd want, not sure if necessary though

Reply:
This pagesize value was already there and a decision that was made when making this component, since it is a constant defined in this file for calls made from this component I don't really see a problem with this and personally I would prefer this.